docker/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:16.04
+
+# Update
+RUN apt-get update && apt-get dist-upgrade -y
+
+# Install the necessary packages.
+RUN apt-get install -y \
+    supervisor apache2 git php7.0 libapache2-mod-php7.0 php7.0-cli \
+    php7.0-mysqlnd php7.0-xml php7.0-mbstring php7.0-curl php7.0-zip
```

Comment:
Ah it looks like ubuntu16.04 was clever enough to fix my mistake when I said to install php7.0-mysqlnd and that's why it worked before I submitted. At least now we have the correct package name, and know why it's there:

![selection_074](https://cloud.githubusercontent.com/assets/5709838/26422400/a27473b2-40c1-11e7-8b4e-aacd028f9d1a.png)